docs/getting_started/create_expectations.rst

```diff
@@ -89,6 +89,50 @@ The argument ``expectation_suite_name`` specifies the name of the expectation su
 
 .. image:: ../images/get_batch.jpg
 
+
+If you want to validate data in Pandas Dataframes or in Spark Dataframes:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Quick nit since I just had to make a change in a different PR minutes ago: turns out tilde isn't an accepted title format. Who knew? Apparently keflavich (https://stackoverflow.com/questions/24504301/what-does-title-level-inconsistent-mean)